apps/web/pages/apps/[slug]/setup.tsx

```diff
@@ -34,7 +34,7 @@ export default function SetupInformation(props: InferGetStaticPropsType<typeof g
 
 export const getStaticPaths = async () => {
   const appStore = await prisma.app.findMany({ select: { slug: true } });
-  const paths = appStore.filter((a) => a.slug in AppSetupPageMap).map((app) => app.slug);
+  const paths = appStore.filter((a) => a.slug in AppSetupMap).map((app) => app.slug);
```

Comment:
@zomars changing this was enough, to just rely on defined App Setup pages map and not App Setup pages with static props map, let me know your thoughts

Reply:
@leog this will break zapier setup I'm not mistaken